### DISCUSSION TITLE
Keep Godot 3 scene import settings in Godot 4 as an option

### DISCUSSION THREAD

**Discussion by joeyeroq** [godot-extended-libraries/godot-ideas#26] [General]
I wanted to test godot 4 and to my surprise/dismay found out that godot 4 removed import default settings .mesh, .material and .anim files.
I find it very handy to put textures, materials, meshes and animations in seperate folders.
In Godot 4 all data types default to extension .tres in stead of extensions that tell what that data is.
Now with the new import settings, you have to perform multiple actions to get some of the same results.
For mesh objects it is the worst.
For example if the .glb file exported from blender is named 'some_character_version_04.glb', a mesh object is named body_part_01' and it's mesh data is named 'Cube.001' the resulting imported mesh file is named some_character_version_04_Cube001.tres', in Godot 3 it would be 'body_part_01.mesh'.
Please keep the godot 3 import settings as an (additional) option.
![godot4_import](https://user-images.githubusercontent.com/70452108/140183026-40faeadb-4df8-48ab-a7bd-e0b739eea285.jpg)

**Comment by Calinou** [comment]
Adding more text resource extensions such as `.tmaterial` or `.tmat` has been discussed, but it may only be implemented after 4.0 is released due to time constraints.

Text files are preferred over binary files since they are more friendly to version control systems like Git.

The rest of this issue seems to be a duplicate of https://github.com/godotengine/godot/issues/54922.
